plugins/yallah/UnityCode/vsm/VSMRemoteConnector.cs

```diff
@@ -0,0 +1,422 @@
+﻿using System.Collections;
```

Comment:
Why is this c# code in the scenemaker repo?

Reply:
This is the corresponding client-side code that communicates with this plugin.
It is tested to work with this version.
Should I rather remove it a put a link to the YALLAH repository?